plugins/MySQL.cpp

```diff
@@ -81,6 +81,33 @@ string MySQLPacket::serializeHandshake() {
     SAppend(handshake.payload, &connectionID, 4); // connection_id
     handshake.payload += (string) "xxxxxxxx";     // auth_plugin_data_part_1
     handshake.payload += lenEncInt(0);            // filler
+
+    static const unsigned CLIENT_LONG_PASSWORD = 0x00000001;
```

Comment:
The codebase doesn't use `static const unsigned` anywhere else.  Don't use `static` or `const` unless strictly needed (which isn't the case here), and use `uint32_t` as done elsewhere in this file.  Basically, please make this code look like the other code in the codebase.